src/main/java/gg/playit/minecraft/PlayitBukkit.java

```diff
@@ -239,14 +249,32 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                         log.warning("failed to create guest secret: " + e);
                         sender.sendMessage("error: " + e.getMessage());
                     } catch (IOException e) {
-                        log.severe("failed to create guest secret: " + e);
+                        log.error("failed to create guest secret: " + e);
                     }
                 }).start();
 
                 return true;
             }
         }
 
+        if (args.length > 0 && args[0].equals("logging")) {
+            if (args.length > 1 && args[1].equals("enable")) {
+                getConfig().set(CFG_DEBUG_LOGGING, true);
```

Comment:
Do we want to save the config after updating the variable?

Reply:
I don't really see whats inherently wrong with doing so, but I moved it to the `onDisable` method if that helps.

Reply:
Implementation is fine, the question if we want to call `saveConfig();` like https://github.com/playit-cloud/playit-minecraft-plugin/blob/a7c5908ac0b8ae7bc7e6d4411788d6f4545e91d2/src/main/java/gg/playit/minecraft/PlayitBukkit.java#LL188C25-L188C38 to make the change persist across server restarts.

Reply:
I think it probably should persist across restarts but I could see an argument for it not to. Not sure if this was an intentional choice or not.

Reply:
> I think it probably should persist across restarts

In my opinion it should. Why should you want to disable debug logs for just one session?